src/TLE.ts

```diff
@@ -1280,61 +1273,61 @@ export class Token {
     }
 
     public static createLeftParenthesis(startIndex: number): Token {
-        return Token.create(TokenType.LeftParenthesis, startIndex, "(");
+        return new Token(TokenType.LeftParenthesis, startIndex, "(");
     }
 
     public static createRightParenthesis(startIndex: number): Token {
-        return Token.create(TokenType.RightParenthesis, startIndex, ")");
+        return new Token(TokenType.RightParenthesis, startIndex, ")");
     }
 
     public static createLeftSquareBracket(startIndex: number): Token {
-        return Token.create(TokenType.LeftSquareBracket, startIndex, "[");
+        return new Token(TokenType.LeftSquareBracket, startIndex, "[");
     }
 
     public static createRightSquareBracket(startIndex: number): Token {
-        return Token.create(TokenType.RightSquareBracket, startIndex, "]");
+        return new Token(TokenType.RightSquareBracket, startIndex, "]");
     }
 
     public static createComma(startIndex: number): Token {
-        return Token.create(TokenType.Comma, startIndex, ",");
+        return new Token(TokenType.Comma, startIndex, ",");
     }
 
     public static createPeriod(startIndex: number): Token {
-        return Token.create(TokenType.Period, startIndex, ".");
+        return new Token(TokenType.Period, startIndex, ".");
     }
 
     public static createWhitespace(startIndex: number, stringValue: string): Token {
         assert(stringValue);
         assert(1 <= stringValue.length);
 
-        return Token.create(TokenType.Whitespace, startIndex, stringValue);
+        return new Token(TokenType.Whitespace, startIndex, stringValue);
     }
 
     public static createQuotedString(startIndex: number, stringValue: string): Token {
         assert(stringValue != null);
         assert(1 <= stringValue.length);
         assert(Utilities.isQuoteCharacter(stringValue[0]));
```

Comment:
>assert(Utilities.isQuoteCharacter(stringValue[0])); [](start = 8, length = 51)

Nit: Unrelated to this PR, why not check the last character as well?

Reply:
The token will be created even if the string is malformed, e.g. no ending quote.  Not sure that's the optimal design, but it's the way it's always been.